renovate.json

```diff
@@ -27,112 +19,62 @@
     "prCreation": "immediate",
     "prPriority": 5
   },
-  "major": {
-    "labels": [
-        "Dependencies",
-        "Renovate"
-    ],
-    "prCreation": "not-pending",
-    "prPriority": 0
-  },
-  "prHourlyLimit": 1,
+  "prHourlyLimit": 10,
+  "enabledManagers": ["dockerfile", "docker-compose", "npm", "composer", "github-actions", "gomod", "terraform"],
   "packageRules": [
     {
       "description": [
-        "composer and npm Minor and Patch updates",
-        "Make PRs after updated package has been stable for 3 days.",
-        "These might be automerged once we're comfortable with Renovate."
-      ],
-      "automerge": false,
-      "groupName": "Patch & Minor Updates (3 day stability)",
-      "groupSlug": "all-minor-patch-updates",
-      "labels": [
-        "Dependencies",
-        "Renovate"
+        "No major version updates"
       ],
-      "matchDatasources": [
-        "composer",
-        "npm"
-      ],
-      "matchUpdateTypes": [
-        "minor",
-        "patch"
-      ],
-      "prCreation": "immediate",
-      "prPriority": 4,
-      "stabilityDays": 3
+      "matchUpdateTypes": ["major"],
+      "enabled": false
     },
     {
       "description": [
-        "Docker Major Dependency Exclusions",
-        "postgres - Excluded from updates to allow us to match major versions to the deployed version in AWS",
-        "php - No migration to PHP 9 until we've got 8.2 deployed"
-      ],
-      "matchUpdateTypes": [
-        "major"
-      ],
-      "matchDatasources": [
-        "docker"
-      ],
-      "matchPackageNames": [
-        "postgres",
-        "php"
+        "No updates to PHP PDF libraries (too sensitive and risky)"
       ],
+      "matchUpdateTypes": ["major", "minor", "patch"],
+      "matchManagers": ["composer"],
```

Comment:
Is `matchDatasources` different and how?

Reply:
I found it easier to use managers, as it's difficult to work out what the datasources are. I honestly don't know the difference.